### PR TITLE
Add topic: digital-public-goods

### PR DESCRIPTION
**Pull request by evilaliv3**
### Please confirm this pull request meets the following requirements:

- [X] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [X] I am not the sole author or employee of a company who created either the topic I am modifying/adding or the collection entry I am modifying/adding.

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [X] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Curating a new topic or collection

- [X] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [X] My folder contains a `*.png` image (if applicable) and `index.md`
- [X] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

I consider this section should be curated in order to highlight the efforts of the communities trying to promote the culture of opensource and promoting the creation of public digital tools for civic purposes.

The main references on this topic is: https://en.wikipedia.org/wiki/Digital_public_goods

Many are already the software and repositories that already adopts the tag: [digital-public-goods](https://github.com/topics/digital-public-goods)

\cc @ricardomiron @BolajiAyodeji

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by BolajiAyodeji** [comment]
Thanks for submitting this PR @evilaliv3! +1.

I'd suggest a few changes below to the copy to fix some typos and reflect a more detailed [definition](https://digitalpublicgoods.net/digital-public-goods/) of digital public goods. I also added an additional alias which is popularly used too (I'm guessing that field can take more than one). Would you be open to updating your PR? Thank you!

```
---
aliases: digital-public-good, dpg
display_name: Digital Public Good
short_description: A digital public good is an open-source digital solution that does no harm and addresses global challenges for public benefit.
topic: digital-public-goods
wikipedia_url: https://en.wikipedia.org/wiki/Digital_public_goods
---
According to the UN Secretary General’s Roadmap for Digital Cooperation, a Digital Public Good (DPG) is an open-source software, open data, open AI systems, or open content collections that adhere to privacy and other applicable laws and best practices, do no harm, and help attain the Sustainable Development Goals (SDGs). To be classified as a DPG, a digital solution must conform to the nine indicators of the [DPG Standard](https://digitalpublicgoods.net/standard). The [Digital Public Goods Alliance](https://digitalpublicgoods.net) maintains a [registry](https://digitalpublicgoods.net/registry) of digital solutions officially vetted and recognized as digital public goods.
```

**Comment by BolajiAyodeji** [comment]
> Thank you for your valuable feedback @BolajiAyodeji
> 
> I've updated it!

Thank you more!

**Comment by ricardomiron** [comment]
This curated list can also be very complementary to the existing GitHub effort on [For Good First Issue](https://forgoodfirstissue.github.com/) which also promotes recognized digital public goods.

**Comment by evilaliv3** [comment]
Thank you @ricardomiron , i just opened one request to add globaleaks as well: https://github.com/github/forgoodfirstissue/issues/192

File: topics/digital-public-goods/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+aliases: digital-public-good, dpg
+display_name: Digital Public Good
+short_description: A digital public good is an open source digital solution that does no harm and addresses global challenges for public benefit.
+topic: digital-public-goods
+wikipedia_url: https://en.wikipedia.org/wiki/Digital_public_goods
+---
+According to the UN Secretary General’s Roadmap for Digital Cooperation, a Digital Public Good (DPG) is an open source software, open data, open AI systems, or open content collections that adhere to privacy and other applicable laws and best practices, do no harm, and help attain the Sustainable Development Goals (SDGs). To be classified as a DPG, a digital solution must conform to the nine indicators of the [DPG Standard](https://digitalpublicgoods.net/standard). The Digital Public Goods Alliance maintains a [registry](https://digitalpublicgoods.net/registry) of digital solutions officially vetted and recognized as digital public goods.
```
Comment (TheGiraffe3) [inline topics/digital-public-goods/index.md:8]: Maybe use a straight quote?
```suggestion
According to the UN Secretary General's Roadmap for Digital Cooperation, a Digital Public Good (DPG) is an open source software, open data, open AI systems, or open content collections that adhere to privacy and other applicable laws and best practices, do no harm, and help attain the Sustainable Development Goals (SDGs). To be classified as a DPG, a digital solution must conform to the nine indicators of the [DPG Standard](https://digitalpublicgoods.net/standard). The Digital Public Goods Alliance maintains a [registry](https://digitalpublicgoods.net/registry) of digital solutions officially vetted and recognized as digital public goods.
```

**Review by tomthorogood — APPROVED** [review]

**Comment by evilaliv3** [comment]
Thank you @tomthorogood 

Do you know when changes will be published? Long time ago we worked on other categories like #whistleblowing but they seems still not published. thank you!

**Comment by tomthorogood** [comment]
@evilaliv3 let me see what i can do about that for you, thanks for letting me know!

**Comment by tomthorogood** [comment]
It looks like digital-public-goods was imported successfully. We're working through an error in the import of collections, however.

**Comment by evilaliv3** [comment]
Oh! Is there a ticket for reference?


Maybe we could try to implement a test to verify the publish count met the directories count?

**Pull request merged**